### PR TITLE
Test branch

### PR DESCRIPTION
**Pull request by alimulondo**
What does this PR do?

This PR Implements the test cases of the api.

Description of Task to be completed?
View questions endpoint tested
Answer to question tested
View single question tested

.

### CODE REVIEW

**Review by hadijahkyampeire — APPROVED** [review]
LGTM

File: tests/Test_base.py [hunk 1]
```diff
@@ -32,8 +32,34 @@ def test_question_content(self):
...
+        self.assertEqual(result.status_code, 200)      
+
+    
+    def test_view_single_questions_two(self):
+        """test viewing single question with right id."""
+        questionId = 100
+
+        result = self.client.get('/api/v1/questions/{}'.format(questionId), content_type = 'application/json')
+        
+        self.assertEqual(result.status_code, 200)
```
Comment (JoyLubega) [inline tests/Test_base.py]: Avoiding testing for status codes only. Test for data

**Review by JoyLubega — CHANGES_REQUESTED** [review]

**Review by JoyLubega — CHANGES_REQUESTED** [review]
Make your Readme detailed, its should include project setup

**Comment by alimulondo** [comment]
I have effected the changes

**Comment by alimulondo** [comment]
Challenge two completed

**Pull request merged**